### PR TITLE
Add complexity to participant password generation logic.

### PR DESCRIPTION
**Pull request by eschlange**
[#113186901]

### CODE REVIEW

File: app/assets/javascripts/think_feel_do_dashboard/participants/_form.js [hunk 1]
```diff
@@ -8,14 +8,19 @@
...
     var password = [];
 
     for (var i = 0; i < PASSPHRASE_LENGTH; i++) {
-      password.push(words()[getRandomInt(0, words().length)]);
+      var passwordFragment = words()[getRandomInt(0, words().length)];
+      password.push(capitalizeFirstLetter(passwordFragment));
     }
 
-    this.password = password.join(' ');
+    this.password = password.join(getRandomInt(0,9));
```
Comment (michaelwehrley) [inline app/assets/javascripts/think_feel_do_dashboard/participants/_form.js]: What is going on here?  Thanks!

Comment (eschlange) [inline app/assets/javascripts/think_feel_do_dashboard/participants/_form.js]: This join method is adding a randomized integer in between each of the randomized words in the password array.

Comment (michaelwehrley) [inline app/assets/javascripts/think_feel_do_dashboard/participants/_form.js]: I love it now.  Thank you!

**Comment by michaelwehrley** [comment]
I approve when **Hakiri** passes.  Thanks!

**Comment by eschlange** [comment]
Hakiri passed, merging in.

**Pull request merged**